ci-automation/garbage_collect.sh

```diff
@@ -269,6 +271,6 @@ function _garbage_collect_impl() {
     echo
 
     source ci-automation/garbage_collect_github_ci_sdk.sh
-    garbage_collect_github_ci
+    garbage_collect_github_ci 1 "${min_age_days}"
```

Comment:
Just one SDK build to keep? Asking to make sure, especially that the default was 20. I do realize that probably more will be kept until they are older than 14 days or so.

Reply:
The "1" just means that at least one is kept; for everything else the 14-day limit will apply.